man/Makefile.am

```diff
@@ -119,7 +119,7 @@ nodist_man_MANS = \
 
 dist_noinst_DATA += $(dist_noinst_man_MANS) $(dist_man_MANS)
 
-SUBSTFILES += $(nodist_man_MANS)
+SUBSTFILES += $(nodist_man_MANS) %D%/man8/zfs_prepare_disk.8
```

Comment:
With https://github.com/openzfs/zfs/pull/15979 now merged this hunk should be dropped.

Reply:
Hi Brian, updated, thanks!